.github/workflows/lifecycle.yml

```diff
@@ -17,4 +17,4 @@ permissions:
 
 jobs:
   lifecycle:
-    uses: kumahq/.github/.github/workflows/wfc_lifecycle.yml@main
+    uses: ./.github/workflows/wfc_lifecycle.yml
```

Comment:
this get's synced right? so there won't be `./.github/workflows/wfc_lifecycle.yml` in `kumahq/kuma` for example

Reply:
To me it doesn't get synced but I'm not sure. I think it's just written independently